go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -793,6 +793,15 @@ func (s *VtctldServer) GetVSchema(ctx context.Context, req *vtctldatapb.GetVSche
 	}, nil
 }
 
+// ApplyVSchema is part of the vtctlservicepb.VtctldServer interface.
+func (s *VtctldServer) ApplyVSchema(ctx context.Context, req *vtctldatapb.ApplyVSchemaRequest) (*vtctldatapb.ApplyVSchemaResponse, error) {
+	if err := s.ts.SaveVSchema(ctx, req.Keyspace, req.VSchema); err != nil {
```

Comment:
looks like you're missing the ["keyspace existence" check](https://github.com/vitessio/vitess/blob/61e812af4546c7609b9359f0bd081f3f498f7844/go/vt/vtctl/vtctl.go#L3231-L3236)

Reply:
done